data/json/npcs/refugee_center/surface_refugees/NPC_Dana_Nunez.json

```diff
@@ -490,8 +490,8 @@
   {
     "id": "landough_calrisean",
     "type": "GENERIC",
-    "copy-from": "sourdough_starter",
-    "looks_like": "sourdough_starter",
+    "copy-from": "sourdough_starter_uncovered",
+    "looks_like": "sourdough_starter_uncovered",
```

Comment:
You should copy the values of the original `sourdough_start` item here because according to the description it is still a jar containing a sourdough starter, and the sourdough it contains "doesn't look like it could be readily salvaged as a sourdough culture".

You may also migrate it like the other items, so you can eat the sourdough and steal the jar, but that's harder to implement (for example the original description about the jar has nowhere to go, and the quest associated with this item might need to be updated.)

Reply:
Thanks for the hint! Done.

I have left `"copy-from": "sourdough_starter_uncovered"` in place for things like `price`, `symbol`, `category` which were not changed between `sourdough_starter` and `sourdough_starter_uncovered`